WordPress/Classes/ViewRelated/Blaze/Webview/BlazeFlowCoordinator.swift

```diff
@@ -108,4 +113,27 @@ import UIKit
         let campaignsViewController = BlazeCampaignsViewController(blog: blog)
         viewController.navigationController?.pushViewController(campaignsViewController, animated: true)
     }
+
+    /// Used to display the blaze campaign details web view.
+    /// - Parameters:
+    ///   - viewController: The view controller where the web view should be presented in.
+    ///   - source: The source that triggers the display of the blaze web view.
+    ///   - blog: `Blog` object representing the site that is being blazed
+    ///   - campaignID: `Int` representing the ID of the campaign whose details is being accessed.
+    @objc(presentBlazeCampaignDetailsInViewController:source:blog:campaignID:)
+    static func presentBlazeCampaignDetails(in viewController: UIViewController,
+                                            source: BlazeSource,
+                                            blog: Blog,
+                                            campaignID: Int) {
+        let blazeViewController = BlazeWebViewController(delegate: nil)
+        let viewModel = BlazeCampaignDetailsWebViewModel(source: source,
+                                                         blog: blog,
+                                                         campaignID: campaignID,
+                                                         view: blazeViewController)
+        blazeViewController.viewModel = viewModel
+        let navigationViewController = UINavigationController(rootViewController: blazeViewController)
+        navigationViewController.overrideUserInterfaceStyle = .light
+        navigationViewController.modalPresentationStyle = .formSheet
+        viewController.present(navigationViewController, animated: true)
```

Comment:
I suggest pushing it in the navigation stack:

https://github.com/wordpress-mobile/WordPress-iOS/assets/1567433/e931d245-2fc2-4215-b3fb-178b0f6af4b2

The "Done" button will need to go.

Reply:
In the Figma design (Ref: 1Osrn9gb2xLaKG6lIft3WC-fi-1461_9777), it's presented in a modal. But yes, that's a good idea, especially cause of the chevron. 

@osullivanchris What do you think?

Reply:
> The only advantage I see of the modal is that its a bit clearer that you're going out of the app to a webview
Great point, especially given the design differences and lack of dark mode support on the web. I opened a thread to discuss it: p1687806482680829-slack-C04LJFS1G5P.